gadget/params.c

```diff
@@ -208,6 +208,12 @@ create_gadget_parameter_set()
     param_declare_double(ps, "BlackHoleFeedbackRadiusMaxPhys", OPTIONAL, 0, "If set, the physical radius at which the black hole feedback energy is deposited. When both this flag and BlackHoleFeedbackRadius are both set, the smaller radius is used.");
     param_declare_int(ps,"WriteBlackHoleDetails",OPTIONAL, 0, "If set, output BH details at every time step.");
 
+    param_declare_int(ps,"BH_DynFrictionMethod",OPTIONAL, 0, "If set to non-zero, dynamical friction is applied through this method.");
```

Comment:
Could you add some more documentation here and in blackhole.c about what each BH_DynFrictionMethod value means? It distinguishes between DynFrictionMethod = 1 and  DynFrictionMethod = 3 in a few places.